media-gfx/curaengine/files/curaengine-2.3.1-remove-gcodeplannertest.patch

```diff
@@ -0,0 +1,225 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 47dcd2d1..c2316d68 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -110,9 +125,6 @@ set(engine_SRCS # Except main.cpp.
+ )
+ 
+ # List of tests. For each test there must be a file tests/${NAME}.cpp and a file tests/${NAME}.h.
+-set(engine_TEST
+-    GCodePlannerTest
+-)
+ set(engine_TEST_INFILL
+ )
+ set(engine_TEST_UTILS
+diff --git a/tests/GCodePlannerTest.cpp b/tests/GCodePlannerTest.cpp
+deleted file mode 100644
+index 4c0493ed..00000000
+--- a/tests/GCodePlannerTest.cpp
```

Comment:
Do you need to actually remove the file? It just makes the diff huge for no gain to the end user.